daemon/kmd/wallet/driver/ledger_hid.go

```diff
@@ -30,6 +30,15 @@ type LedgerUSB struct {
 	hiddev *hid.Device
 }
 
+// LedgerUSBError is a wrapper around the two-byte error code that the Ledger
+// protocol returns.
+type LedgerUSBError uint16
+
+// Error satisfies builtin interface `error`
+func (err LedgerUSBError) Error() string {
+	return fmt.Sprintf("unexpected status %x", err)
```

Comment:
nit: change it to `Exchange: unexpected status 0x%x` so that we'll know where this issue came from as well as making it easier to decode.

Reply:
Good point.  Pushed; can you re-approve?